gui/slick/views/home.mako

```diff
@@ -781,12 +776,13 @@ $(document).ready(function(){
             <span class="visible-print-inline">${download_stat}</span>
         </td>
 
-        <% show_size = sickbeard.helpers.get_size(curShow._location) %>
-        <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        ## <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        ## <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        <td align="center" data-show-size="0">Disabled!</td>
```

Comment:
I guess the easiest way would be to add `class="hidden"` (maybe check that this is the correct Bootstrap class) [here](https://github.com/SiCKRAGETV/SickRage/pull/2356/files#diff-ce131b2d456b1eaed7f0dba1e738d4e6R609), [here](https://github.com/SiCKRAGETV/SickRage/pull/2356/files#diff-ce131b2d456b1eaed7f0dba1e738d4e6R625) and on your "Disabled" cell (which can now be empty).